stable/kommander/templates/grafana/opsportal-username-secret.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       containers:
         - name: kubectl
           # --export flag is deprecated so we need to stick with this kubectl version
-          image: bitnami/kubectl:1.16.2
+          image: bitnami/kubectl:1.19.7
           imagePullPolicy: IfNotPresent
           command:
             - /bin/sh
```

Comment:
actually we don't need to use to `--export` here (line 27), I think maybe it was left over from a precious implementation that was copying the entire secret over.

Reply:
pushed the fix to remove https://github.com/mesosphere/charts/pull/962/commits/288d9106540ff3a5b4443f28dbcef9682236ff5c